merge.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# You can use the Makefile to create the merged output but that requires an install of npm and jq
```

Comment:
@gabywh I find this comment slightly misleading. Instead?:

# This shell script merges the various sub schemas into the DCC.combined-schema.json, using Python.
# The same task is performed by the Makefile which additionally relies on jq to reformat the combined schema, and on ajv to validate the merged schema.

Reply:
Please be specific. What do you find misleading?

Your re-formulation misses the point completely that the script was meant to be something of a lightweight and somewhat hard-coded version of what was in the Makefile - explicitly for people who don't have all the node / npm / ajv / jq tooling set-up - a lightweight entry point for the not too technical end-user.

Your reformulation misses this core point completely.

Reply:
Also missing completely is any mention (in README or otherwise) of needing to install all that tooling. Mentioning this core point in comments in a script doesn't alleviate that.

Happy to add that documentation to the README, though.

Reply:
Yes instructions for end-user install "missing" - pirmarily because it seems to vary widely amongst implementers.

That said, feel free to add sth if you wish, along the lines of:
 * node 
 * npm
 * ajv
 * jq
 
for Windows, Mac and *nix users, which would be a basic set of dev install.

Reply:
I still didn't see an explanation of what you find misleading? Perhaps you could elaborate?

Reply:
`package.json` is Git-ignore'd, which is unusual given the reliance on Node/NPM. Is there a particular reason for that?

Reply:
Also this: people install what they will - not everyone will be using JS and associated tooling / infra so it is something of an arguable point solely on that basis whether we want to include such files as `package.json` in the repo. I'd say not for that reason alone. A further reason not to include `package.json` in the repo it that you can generate it, so as it is an output / generated file it also doesn't belong in the source repo anyway.

Reply:
People used to JS infra will very much *expect* a non-generated/-Git-ignore'd `package.json`, and be confused without it - at least without installation instructions <== fixed per #8154614

Reply:
> People used to JS infra will very much _expect_ a non-generated/-Git-ignore'd `package.json`, and be confused without it 

not sure I follow - if we .gitignore it and people used to JS infra know you can generate it, then why would they be expecting it and be confused without it? I just generated it myself, took a couple of seconds, simply wasn't an issue.

> - at least without installation instructions <== fixed per #8154614

and no I didn't need installation instructions either - but no harm in supplying them

Reply:
This repo isn't a real NPM package - we just need `ajv` - so a proper `package.json` is not necessary.

Reply:
but you can generate it, for sure, in fact it will complain if it isn't there - hence .gitignore, hence generated output, hence not in the repo hence not confusing etc - as above

I still haven't seen an explanation of what you claimed to be misleading...